### PR TITLE
New lifestyles - small improvements

### PR DESCRIPTION
**Pull request by MemeWizard**
### Changelog:
- [x] Fixed characters not spending acquired vanilla perk points
- Characters with physical lifestyle traits are now less likely to pick magic lifestyles

### Developer changelog:
- Replaced multiple `can_level` triggers with one
- Deleted superfluous `MAGIC_magic_lifestyle_tree_TREE_finished_trigger` trigger
- Merged some physical and magical files
- Renamed a lot of things (like scripted values) to better reflect what they are used for
- Added basic `auto_selection_weight` for physical focuses and perks

## Tests:
- [x] There are no errors in `wc` files in `Documents\Paradox Interactive\Crusader Kings III\logs\error.log` except `portrait_decals.cpp:101`
- [x] The mod takes less than 5.5 GB in the Task Manager (Windows)

### CODE REVIEW

File: common/scripted_triggers/00_lifestyle_triggers.txt [hunk 1]
```diff
@@ -82,121 +82,47 @@ can_start_new_lifestyle_tree_trigger = {
 		has_perk = avaricious_perk
 	}
 
-	## Warcraft - Mod triggers
+	## Warcraft
```
Comment (zumbak04) [inline common/scripted_triggers/00_lifestyle_triggers.txt:85]: Why `diplomacy_lifestyle_tree_1_finished_or_not_started_trigger`, etc. aren't added in this file?

File: common/decisions/test_decision.txt [hunk 2]
```diff
@@ -6,7 +6,7 @@
 	}
```
[the remark this answers is not in the document]
Comment (MemeWizard) [reply common/decisions/test_decision.txt:6]: ooops

[the remark this answers is not in the document]
Comment (MemeWizard) [reply common/decisions/test_decision.txt:6]: I guess I forgot to revert a commit, should be fine now

File: common/scripted_triggers/00_lifestyle_triggers.txt (hunk 1 above)
Comment (MemeWizard) [reply common/scripted_triggers/00_lifestyle_triggers.txt]: Done

**Review by zumbak04 — APPROVED** [review]
LGTM!

**Pull request merged**